baton-maven-plugin/src/test/resources/specifications/util/pom-utils.feature

```diff
@@ -0,0 +1,18 @@
+@baton
+Feature: Baton utilities can be used to assist with migration logic pertaining to Maven POM files
+  Scenario: I can leverage POM helper utilities to delete an element from a pom file
+    Given a pom file with existing elements
+    When I write a deletion for an existing element
+    Then the existing element should not be present in the pom file
+
+  Scenario: I can leverage POM helper utilities to replace an element in a pom file
+    Given a pom file with existing elements
+    When I write a replacement for an existing element
+    Then the new element should be present in the pom file in place of the existing element
+
+  Scenario: I can leverage POM helper utilities to insert an element in a pom file
+    Given a pom file with existing elements
+    When I write an insertion for a new element
+    Then the new element should be present in the pom file
+
+  Scenario: I can leverage POM helper utilities to write to a POM file
```

Comment:
Q: Should this scenario have an steps to it?

Reply:
Nope! I included Scenarios but no steps for functionality that was too straightforward to need unit tests for. If you think it's an unnecessary practice, I can remove these kinds of scenarios.